api/RESTful API/Posts/postRouter.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const router = express.Router();
+const userDB = require('../../../data/helpers/postDb');
```

Comment:
You are calling variable `userDB` but importing `postDB` which is clearly a mistype.